cmd/subctl/verify.go

```diff
@@ -132,6 +133,8 @@ func addVerifyFlags(cmd *cobra.Command) {
 	cmd.Flags().UintVar(&verifyFlags.PacketSize, "packet-size", 3000, "set packet size used in TCP connectivity tests")
 	cmd.Flags().BoolVar(&verifyFlags.SkipConnectorSrcIPCheck, "skip-src-ip-check", false,
 		"skip source IP verification for connector pod traffic")
+	cmd.Flags().BoolVar(&verifyFlags.SkipIntraClusterRoutingTests, "skip-intra-cluster-routing-tests", false,
+		"skip tests that verify intra-cluster routing component")
```

Comment:
I think we should keep the wording consistent with the join flag, ie use the term _connectivity_:

```suggestion
	cmd.Flags().BoolVar(&verifyFlags.SkipIntraClusterRoutingTests, "skip-intra-cluster-connectivity-tests", false,
		"skip tests that verify intra-cluster connectivity functionality")
```

Reply:
@tpantelis `intra-cluster-connecitivity` should be enough, don't need `functionality`